src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/Example1.java

```diff
@@ -10,18 +10,43 @@
 // xdoc section -- start
 // violation 5 lines below '<p> tag should be preceded with an empty line'
 /**
- * No tag (ok).
+ * No tag
  *
- * <p>Tag immediately before the text (ok).
- * <p>No blank line before the tag (violation).
+ * <p>Tag immediately before the text
+ * <p>No blank line before the tag
  *
  * <p>
- * New line after tag (violation).
+ * New line after tag
  *
- * <p> Whitespace after tag (violation).
+ * <p> Whitespace after tag
  *
  */
 // violation 3 lines above 'tag should be placed immediately before the first word'
+
 public class Example1 {
+  // violation 4 lines below '<p> tag should not precede HTML block-tag '<pre>''
+  /**
+   * No tag
+   *
+   * <p>
+   * <pre>item 1</pre>
```

Comment:
Can you also add an example of a inline tag preceded by `<p>` tag?

```
<p><b>this is okay, B tag is an inline tag</b></p>
```

Reply:
@akankshat05, is this done?

Please respond done to each review items it helps a lot.

Reply:
This is done.